### ISSUE TITLE
Keep up the good work

### ISSUE THREAD

**Issue by shoyrock** [FlorianEagox/weeablind#7]
I love this project! Keep up the excellent work. I've wanted something like this for years, but I don't have the skills. I wish I had the skill to help you work on this. There are so many good animes out there that I have been unable to watch because they never got a dub or will never get one. You're doing God's work.

**Comment by FlorianEagox** [comment]
Thank you so much! Let me know if you have any problems or feedback!

**Comment by shoyrock** [comment]
I haven’t tested it as yet, but I’ll let you know if I need any help if the need arise

**Comment by Jellybit** [comment]
Yes, this is an amazing project! Thank you for sharing it, and I hope you like the results.

**Comment by shoyrock** [comment]
I found this I don’t know if y’all know about it https://github.com/2noise/ChatTTS
